sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -465,56 +484,70 @@ def get_windowing(self, inputs):
     return inputs[0].windowing
 
   def __rrshift__(self, label):
+    # type: (str) -> _NamedPTransform[InT, OutT]
     return _NamedPTransform(self, label)
 
   def __or__(self, right):
+    # type: (PTransform[InT, OutT], PTransform[OutT, T]) -> _ChainedPTransform[OutT, T]
     """Used to compose PTransforms, e.g., ptransform1 | ptransform2."""
     if isinstance(right, PTransform):
       return _ChainedPTransform(self, right)
     return NotImplemented
 
-  def __ror__(self, left, label=None):
-    """Used to apply this PTransform to non-PValues, e.g., a tuple."""
-    pvalueish, pvalues = self._extract_input_pvalues(left)
-    pipelines = [v.pipeline for v in pvalues if isinstance(v, pvalue.PValue)]
-    if pvalues and not pipelines:
-      deferred = False
+  if not typing.TYPE_CHECKING:
```

Comment:
I wish this was a decorator instead of adding an indent.

Reply:
interesting idea, but how do you decorate an import statement?

Reply:
I don't think you can decorate an import, but this is a method.

Reply:
ah, sorry, I missed the context here.  yeah, that would be a nice feature.  would be worth bringing up at the mypy github repo.

note that this change is to accommodate analyzing user pipelines via the mypy plugin, which  I'm now thinking would be best to separate into another PR.   With some more hacking, I might ultimately be able to avoid this bit of ugliness.